### ISSUE TITLE
Use latest RHBK in nightly pipelines

### ISSUE THREAD

**Issue by trepel** [Kuadrant/testsuite-pipelines#36]
**Summary**

Use latest RHBK (Red Hat Build of Keycloak) in nightly pipelines to test the Authorino integration with Keycloak.

**Comment by averevki** [comment]
I think this issue belongs under the testsuite repo. Pipelines have nothing to do with cluster environment for testsuite. Feel free to assign me :+1:

**Comment by trepel** [comment]
@averevki  Hmm, to be honest I did not think much about it when creating this issue but thinking about it now, testsuite should not be responsible for setting up cluster, kuadrant, tools etc. Testsuite expects all this to be in place before it is triggered, no?

In my mind we aim for pipeline capable of creating a cluster, installing kuadrant with all the dependencies and configuring it, installing tools and other dependencies of the test suite, triggering testsuite, reporting results, cleanup.

**Comment by averevki** [comment]
> @averevki Hmm, to be honest I did not think much about it when creating this issue but thinking about it now, testsuite should not be responsible for setting up cluster, kuadrant, tools etc. Testsuite expects all this to be in place before it is triggered, no?

You are correct, but with tools it doesn't matter what tools there are. They are all, essentially, installed with one command and the issue here would be to make tools compatible with the testsuite. We had some problems with the deployment of different keycloak providers before, some of them use different credentials secret naming, realms and users which are created during tests can have different requirements, the way each of them setup tls, etc.

> In my mind we aim for pipeline capable of creating a cluster, installing kuadrant with all the dependencies and configuring it, installing tools and other dependencies of the test suite, triggering testsuite, reporting results, cleanup.

I guess the nightly pipeline mentions misled me. I believe we have never talked about the new cluster provisioning pipelines, for now. We should create an epic for this.

**Comment by trepel** [comment]
Thanks for explanation. I will transfer the issue as you suggested.

> I guess the nightly pipeline mentions misled me. I believe we have never talked about the new cluster provisioning pipelines, for now. We should create an epic for this.

You're right that this would probably warrants an Epic. So far I just created issues related to cluster provisioning recently.
